assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -58,17 +63,48 @@ const subHeadingText = {
     monthly or one-time contribution today`,
 };
 
-function contribAttrs(isoCountry: IsoCountry): ContribAttrs {
+function contribCtaText(contribType: Contrib): string {
+  switch (contribType) {
+    case 'RECURRING':
+      return 'Contribute with card or PayPal';
+    default:
+      return 'Contribute with credit/debit card';
+  }
+}
```

Comment:
Yes. We're not expecting any other contribution types in the short term I guess!